pkg/Makefile

```diff
@@ -41,6 +41,13 @@ osc-cli-x86_64.zip: osc-cli
 	zip -r osc-cli-x86_64.zip osc-cli
 
 osc-cli:
+	@echo    '==========================================='
+	@echo -e '============= \033[31m  WARNING !!!!! \033[m ============'
+	@echo    '==========================================='
+	@echo -e '\033[5m\033[31mThis rule is deprecated, and most likely broken \033[m'
+	@echo    '==========================================='
+	@echo -e '============= \033[31m \WARNING !!!!! \033[m ============'
+	@echo    '==========================================='
```

Comment:
Why not removing old rule?

Reply:
can we do that in a second time, as this is still  a very diferent way to package the windows cli.